electron{-source,-bin,-chromedriver}: updates

teutat3s opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

https://github.com/electron/electron/releases/tag/v35.7.0
https://github.com/electron/electron/releases/tag/v36.7.0
https://github.com/electron/electron/releases/tag/v37.2.0

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md), [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) and other contributing documentation in corresponding paths.

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

teutat3s committed:
electron_35-bin: 35.6.0 -> 35.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v35.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v35.6.0...v35.7.0

teutat3s committed:
electron-chromedriver_35: 35.6.0 -> 35.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v35.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v35.6.0...v35.7.0

teutat3s committed:
electron_36-bin: 36.6.0 -> 36.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v36.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v36.6.0...v36.7.0

teutat3s committed:
electron-chromedriver_36: 36.6.0 -> 36.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v36.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v36.6.0...v36.7.0

teutat3s committed:
electron_37-bin: 37.1.0 -> 37.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.1.0...v37.2.0

teutat3s committed:
electron-chromedriver_37: 37.1.0 -> 37.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.1.0...v37.2.0

teutat3s committed:
electron-source.electron_35: 35.6.0 -> 35.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v35.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v35.6.0...v35.7.0

teutat3s committed:
electron-source.electron_36: 36.6.0 -> 36.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v36.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v36.6.0...v36.7.0

teutat3s committed:
electron-source.electron_37: 37.1.0 -> 37.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.1.0...v37.2.0

teutat3s commented:
New releases are out ... but I did build `electron_37` with success on `x86_64-linux` today.

TomaSajt commented:
Would it be bad practice to just send minor/patch electron updates to master without completely checking if they build or not?

And in the rare case that the build fails on hydra, we just revert, so the previously cached build can be used until we figure out the issue.

teutat3s committed:
electron_35-bin: 35.7.0 -> 35.7.1

- Changelog: https://github.com/electron/electron/releases/tag/v35.7.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v35.7.0...v35.7.1

teutat3s committed:
electron-chromedriver_35: 35.7.0 -> 35.7.1

- Changelog: https://github.com/electron/electron/releases/tag/v35.7.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v35.7.0...v35.7.1

teutat3s committed:
electron_36-bin: 36.7.0 -> 36.7.1

- Changelog: https://github.com/electron/electron/releases/tag/v36.7.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v36.7.0...v36.7.1

teutat3s committed:
electron-chromedriver_36: 36.7.0 -> 36.7.1

- Changelog: https://github.com/electron/electron/releases/tag/v36.7.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v36.7.0...v36.7.1

teutat3s committed:
electron_37-bin: 37.2.0 -> 37.2.1

- Changelog: https://github.com/electron/electron/releases/tag/v37.2.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.2.0...v37.2.1

teutat3s committed:
electron-chromedriver_37: 37.2.0 -> 37.2.1

- Changelog: https://github.com/electron/electron/releases/tag/v37.2.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.2.0...v37.2.1

teutat3s committed:
electron-source.electron_35: 35.7.0 -> 35.7.1

- Changelog: https://github.com/electron/electron/releases/tag/v35.7.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v35.7.0...v35.7.1

teutat3s committed:
electron-source.electron_36: 36.7.0 -> 36.7.1

- Changelog: https://github.com/electron/electron/releases/tag/v36.7.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v36.7.0...v36.7.1

teutat3s committed:
electron-source.electron_37: 37.2.0 -> 37.2.1

- Changelog: https://github.com/electron/electron/releases/tag/v37.2.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.2.0...v37.2.1

K900 commented:
We should probably revert the updates that point at broken tags.
